core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1002,6 +1047,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    */
   @DeveloperApi
   override def requestExecutors(numAdditionalExecutors: Int): Boolean = {
+    assertNotStopped()
```

Comment:
This is a toss-up, since I'd expect to throw some sort of error after the scheduler backend is stopped, but there are many cases where it's a no-op and doesn't throw an error.  Therefore, I'll remove this, too.